specs.json

```diff
@@ -114,6 +115,7 @@
   "https://wicg.github.io/permissions-revoke/",
   "https://wicg.github.io/portals/",
   "https://wicg.github.io/priority-hints/",
+  "https://wicg.github.io/sanitizer-api/",
```

Comment:
I thought it was memory measure you wanted to promote?

Reply:
It sure was! Copy-and-paste can lead to interesting errors... Let's say I was testing your ability to review pull requests :)